demo/api/src/app.module.ts

```diff
@@ -184,7 +186,7 @@ export class AppModule {
                 FooterModule,
                 PredefinedPagesModule,
                 CronJobsModule,
-                ProductsModule,
+                ProductsModule.forRoot(mailerModule),
```

Comment:
This is not optimal. It would be better to make MailerModule a global module

(I just saw that Niko already mentioned that below)